AnnoyancesFilter/sections/antiadblock.txt

```diff
@@ -249,7 +249,7 @@ skegnessstandard.co.uk#@#.text-ad-links
 ! ai_front anti-adblock
 biopills.net,ptisidiastima.com,medebooks.xyz,re-library.com,mobilereset99.com#%#//scriptlet("abort-on-property-write", "ai_front")
 webdeyazilim.com,biopills.net,ptisidiastima.com,medebooks.xyz,re-library.com,mobilereset99.com##[style*="cursor:"][style*="user-select: none;"][style*="z-index:"]
-webdeyazilim.com#%#//scriptlet("abort-current-inline-script", "String.prototype.charCodeAt", "ai_adb")
+freebulksmsonline.com,webdeyazilim.com#%#//scriptlet("abort-current-inline-script", "String.prototype.charCodeAt", "ai_adb")
```

Comment:
There is no way to skip this window, so move the domain to Base filter - we have a similar rule.
Also don't forget about content blockers - we also have rules for this.